pyaedt/q3d.py

```diff
@@ -709,6 +709,425 @@ def export_matrix_data(
+                coupling_limit_value = "By Value"
+            elif coupling_limit_type == 1:
+                coupling_limit_value = "By Fraction Of Self Term"
+
+            coupling_limits.append(coupling_limit_value)
+
+            if cond_limit is None and coupling_limit_type == 0:
+                cond_limit = "1mSie"
+            elif cond_limit is None and coupling_limit_type == 1:
+                cond_limit = "0.01"
+            elif cond_limit is not None:
+                if decompose_variable_value(cond_limit)[1] not in [
+                    "fSie",
+                    "pSie",
+                    "nSie",
+                    "uSie",
+                    "mSie",
+                    "sie",
+                    "kSie",
+                    "megSie",
+                    "mho",
+                    "perohm",
+                ]:
+                    self.logger.error("Invalid conductance unit.")
+                    return False
+
+            coupling_limits.append("CondLimit:=")
+            coupling_limits.append(cond_limit)
+
+            if cap_limit is None and coupling_limit_type == 0:
+                cap_limit = "1pF"
+            elif cap_limit is None and coupling_limit_type == 1:
+                cap_limit = "0.01"
+            elif cap_limit is not None:
+                if decompose_variable_value(cap_limit)[1] not in ["fF", "pF", "nF", "uF", "mF", "farad"]:
+                    self.logger.error("Invalid capacitance unit.")
+                    return False
+
+            coupling_limits.append("CapLimit:=")
+            coupling_limits.append(cap_limit)
+
+            if ind_limit is None and coupling_limit_type == 0:
+                ind_limit = "1nH"
+            elif ind_limit is None and coupling_limit_type == 1:
+                ind_limit = "0.01"
+            elif ind_limit is not None:
+                if decompose_variable_value(ind_limit)[1] not in ["fH", "pH", "nH", "uH", "mH", "H"]:
+                    self.logger.error("Invalid inductance unit.")
+                    return False
+
+            coupling_limits.append("IndLimit:=")
+            coupling_limits.append(ind_limit)
+
+            if res_limit is None and coupling_limit_type == 0:
+                res_limit = "1ohm"
+            elif res_limit is None and coupling_limit_type == 1:
+                res_limit = "0.01"
+            elif res_limit is not None:
+                if decompose_variable_value(res_limit)[1] not in ["uOhm", "mOhm", "ohm", "kOhm", "megOhm", "GOhm"]:
+                    self.logger.error("Invalid resistance unit.")
+                    return False
+
+            coupling_limits.append("ResLimit:=")
+            coupling_limits.append(res_limit)
+        else:
+            coupling_limit_value = "None"
+            coupling_limits.append(coupling_limit_value)
+
+        # if include_cpp and not [x for x in [include_dcr, include_acr, include_dcl, include_acl, add_resistance] if x]:
```

Comment:
Should we uncomment this section, otherwise `include_cpp` is never used?

Reply:
@MaxJPRey please see PR's description.
I would rather delete this part and always set include_cpp to False.

Reply:
Indeed, I missed the description 😄 .
If you delete this section, maybe you can just had a comment (like the one in the PR description) to explain why it is always False.